autoload/SpaceVim/layers/lang/typescript.vim

```diff
@@ -71,3 +84,15 @@ function! s:on_ft() abort
     endif
   endif
 endfunction
+
+function! s:go_to_def() abort
+  if !SpaceVim#layers#lsp#check_filetype('typescript')
+    if has('nvim')
+      call feedkeys(":TSDef\<CR>")
+    else 
+      call SpaceVim#lsp#go_to_def()
+    endif
+  else
```

Comment:
should not use `SpaceVim#lsp#go_to_def()` when `SpaceVim#layers#lsp#check_filetype('typescript')` return 0. so this fnction should be:

```vim
function! s:go_to_def() abort
  if SpaceVim#layers#lsp#check_filetype('typescript')
     call SpaceVim#lsp#go_to_def()
  else
     TSDef
  endif
endfunction
```

Reply:
emm, `TSDef` command seems only for nvim?

Reply:
yes, it provided by `nvim-typescript` plugin. So does it need any change?